evaluation/swe_bench/README.md

```diff
@@ -63,11 +63,22 @@ then your command would be:
 ./evaluation/swe_bench/scripts/run_infer.sh llm.eval_gpt4_1106_preview HEAD CodeActAgent 10
 ```
 
-**Evaluate on `RemoteRuntime` (alpha)** (contact Xingyao over slack if you want to try this out!)
+### Run Inference on `RemoteRuntime`
+
+This is in limited beta. Contact Xingyao over slack if you want to try this out!
+
+```bash
+# ./evaluation/swe_bench/scripts/run_infer.sh [model_config] [git-version] [agent] [eval_limit] [max_iter] [num_workers] [dataset] [dataset_split]
+SANDBOX_API_KEY="YOUR-API-KEY" RUNTIME=remote EVAL_DOCKER_IMAGE_PREFIX="us-docker.pkg.dev/evaluation-428620/swe-bench-images" \
```

Comment:
```suggestion
ALLHANDS_API_KEY="YOUR-API-KEY" RUNTIME=remote EVAL_DOCKER_IMAGE_PREFIX="us-docker.pkg.dev/evaluation-428620/swe-bench-images" \
```

Reply:
I think the reason for the name `SANDBOX_API_KEY` is that we need to load it up with `api_key` field inside `SandboxConfig`? If we want to modify this, we probably need to change the `api_key` under `SandboxConfig` to `allhands_api_key`, so the env var becomes `SANDBOX_ALLHANDS_API_KEY`?

https://github.com/All-Hands-AI/OpenHands/blob/ee158feb15accef6513a44089496d53aa6d48c1f/openhands/core/config.py#L30

Reply:
What about changing this:

https://github.com/All-Hands-AI/OpenHands/blob/main/evaluation/swe_bench/run_infer.py#L126-L132

to 

```python
        sandbox=SandboxConfig(
            base_container_image=base_container_image,
            enable_auto_lint=True,
            use_host_network=False,
            # large enough timeout, since some testcases take very long to run
            timeout=300,
            api_key=os.environ.get('ALLHANDS_API_KEY', None),
        ),
```

Reply:
Sounds good!